js/src/browser/bytes.js

```diff
@@ -57,7 +57,10 @@ export function copy(source: Uint8Array, target: Uint8Array, targetStart: number
 }
 
 export function slice(buff: Uint8Array, start: number, end: number): Uint8Array {
-  return buff.slice(start, end);
+  // Safari does not have slice on typed arrays.
```

Comment:
Did `slice` copy? Obviously this new code does. My point being, is it worth changing this code to:
```
if (buff.slice) {
  return buff.slice(start, end);
}
// Safari does not have slice on typed arrays.
const v = ...;
return v;
```